test/fixtures/03-async-migrator/03-finalize-async/migrations/m0002-data-copy.async.ts

```diff
@@ -0,0 +1,28 @@
+import { AsyncMigration } from '../../../../../src/migrator/utils';
+
+export = {
+	asyncSql: `\
+        UPDATE "device"
+        SET "note" = "device"."name"
```

Comment:
Should this be setting a different column or table than `test/fixtures/03-async-migrator/03-finalize-async/migrations/m0001-data-copy.async.ts`?
Should it be using the `deviceb` table like in `test/fixtures/03-async-migrator/02-parallel-migrations/migrations/0002-migrate-testb.async.ts`?

Reply:
yeah, thanks